clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,120 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 300}}
+{{$MEDIUM_BACKEND_SIZE := 150}}
+{{$SMALL_BACKEND_SIZE := 10}}
+# adding a fixed value for first version of the test, rate of pod creation not a concern yet.
+{{$ilbQPS := 500}}
+{{$namespaces := 1}}
+#Test
+name: ilbload
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+- name: ILBConstantQPS
+  qpsLoad:
+    averageQps: {{$ilbQPS}}
+steps:
+# Create ILBs
+- measurements:
+  - Identifier: ServiceCreationLatency
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+  - Identifier: WaitForRunningDeployments
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      apiVersion: apps/v1
+      kind: Deployment
+      operationTimeout: 15m
+
+- phases:
+  - namespaceRange:
+      min: 1
+      max: {{$namespaces}}
+    replicasPerNamespace: 1
+    tuningSet: ILBConstantQPS
+    objectBundle:
+    - basename: large-backends-dep
+      objectTemplatePath: dep.yaml
+      templateFillMap:
+        ReplicasMin: {{$LARGE_BACKEND_SIZE}}
+        ReplicasMax: {{$LARGE_BACKEND_SIZE}}
+    - basename: large-backends-service
+      objectTemplatePath: service.yaml
+      templateFillMap:
+        DeploymentBaseName: large-backends-dep
+- phases:
```

Comment:
Do we need a separate step here or can we merge it with the one above?

Reply:
I wanted to keep each deployment + service creation as separate steps so that the latencies are measured individually. don't we need that ?

Reply:
- Currently large deployment is on step while medium and small are in the other.
- What do you mean by the latencies being measured individually? Do you want to create deployment+service, wait until it is available and then proceed with the next one? Or do you want to create all deployments+services at once and log latency for each of them separately?

Reply:
the latter.. create all deployments+services in parallel and log latency for each separately. I put all deployment creation in the same step as phases, looks like the measurement is happening correctly.

Reply:
The ServiceCreationLatency measurement observes services individually. However the summary it's creating is an aggregation of all observed service latencies. Maybe we want to have separate measurement for each type (size) of deployment?

Reply:
I do see logs like this:
ServiceCreationLatency: 3 worst create_to_available latencies: [{test-fcrs0g-1/small-backends-service-0 1m6.427795413s} {test-fcrs0g-1/large-backends-service-0 1m46.162875508s} {test-fcrs0g-1/medium-backends-service-0 2m27.217369777s}]

assumed this means each service creation is measured individually. Is this not the  case?

Reply:
They are measured separately.
However, if you look at ServiceCreationLatency_ilblod_<timestamp>.json in directory specified by the 'report-dir' flag, there will be only an aggregation.